gui/src/main/java/org/jboss/as/console/client/domain/hosts/HostMgmtPresenter.java

```diff
@@ -126,6 +102,7 @@ public PlaceManager getPlaceManager() {
             "opt://server-group=*/system-property=*"},
             recursive = false)
     @SearchIndex(keywords = {"host", "jvm", "group", "server-group", "profile", "socket-binding"})
+//    @RequiredResources(resources = {"/{implicit.host}"})
```

Comment:
`@RequiredResources` is the replacement for `@AccessControl`. Does it make a difference if you replace `@AccessControl` with `@RequiredResources` and add `"/{implicit.host}"`?